gnovm/tests/files/zrealm_crossrealm15.gno

```diff
@@ -0,0 +1,27 @@
+// PKGPATH: gno.land/r/crossrealm_test
+package crossrealm_test
+
+import (
+	"std"
+
+	crossrealm "gno.land/r/demo/tests/crossrealm"
+)
+
+type fooer struct{}
+
+func (fooer) Foo() { println("hello " + std.CurrentRealm().PkgPath()) }
+
+var f *fooer
+
+func init() {
+	f = &fooer{}
+	crossrealm.SetFooer(f)
+	crossrealm.CallFoo()
+}
+
+func main() {
+	print(".")
+}
+
+// Error:
+// gno.land/r/crossrealm_test/main.gno:19:2: name CallFoo not declared
```

Comment:
This error message doesn’t seem intended for this pattern. Consider modifying the code or simply removing the message entirely to eliminate any ambiguity.

Reply:
4b298d5b3